### PR TITLE
Expose docker log driver options

### PR DESCRIPTION
**Pull request by dkoshkin**
Fixes https://github.com/apprenda/kismatic/issues/685, fixes https://github.com/apprenda/kismatic/issues/674

Sample plan file:
```
docker:
  logs:
    driver: json-file
    opts:
      max-file: "1"
      max-size: 50m
```

The values above will be used in the docker config json unless the user overwrites these options.

The `opts` is a `map[string]string` so any option can be set.

### CODE REVIEW

**Comment by dkoshkin** [comment]
This PR is a simplified version of https://github.com/apprenda/kismatic/pulls

File: pkg/install/plan.go [hunk 1]
```diff
@@ -99,6 +99,14 @@ func readDeprecatedFields(p *Plan) {
 }
 
 func setDefaults(p *Plan) {
+	if p.Docker.Logs.Driver == "" {
+		p.Docker.Logs.Driver = "json-file"
+		p.Docker.Logs.Opts = map[string]string{
+			"max-size": "50m",
```
Comment (emmetthitz) [inline pkg/install/plan.go]: Can we change this 50MB or something a little less ambiguous?

Comment (dkoshkin) [reply pkg/install/plan.go]: Thats a docker setting https://docs.docker.com/engine/admin/logging/json-file/

Comment (emmetthitz) [reply pkg/install/plan.go]: That is astonishingly stupid. Alright, never mind then.

**Comment by swade1987** [comment]
So what will be the defaults shipped in KET for `max-file` and `max-size` ?

**Comment by dkoshkin** [comment]
@swade1987 the defaults will be:
```
      max-file: "1"
      max-size: 50m
```

File: pkg/install/plan.go (hunk 1 above)
Comment (alexbrand) [reply pkg/install/plan.go]: @emmetthitz Can you elaborate?

**Comment by alexbrand** [comment]
I think this LGTM. I think the last piece is adding docs about this to our docker.md.

**Comment by swade1987** [comment]
@dkoshkin just an FYI, we normally set `max-size: 3m`

**Comment by dkoshkin** [comment]
@swade1987 lets discuss this today.

**Pull request merged**